pyriemann/utils/kernel.py

```diff
@@ -208,8 +208,9 @@ def kernel(X, Y=None, *, Cref=None, metric='riemann', reg=1e-10):
     Cref : None | ndarray, shape (n, n), default=None
         Reference point for the tangent space and inner product
         calculation. Only used if metric='riemann'.
-    metric : {'euclid', 'logeuclid', 'riemann'}, default='riemann'
-        The type of metric used for tangent space and mean estimation.
+    metric : string, default="riemann"
+        Metric used for tangent space and mean estimation, can be:
+        "euclid", "logeuclid", "riemann".
```

Comment:
I find this more readable than the notation with `{"euclid", "logeuclid", "riemann"}`.
If you have to standardize the documentation, I would stick to this one or to the typescript notation: "euclid" | "logeuclid" | "riemann"